libraries/SITL/SITL.cpp

```diff
@@ -168,10 +178,25 @@ const AP_Param::GroupInfo SIM::var_info[] = {
     // @Description: Opflow data delay
     // @Units: ms
     AP_GROUPINFO("FLOW_DELAY",    36, SIM,  flow_delay, 0),
+    // @Param: ADSB_COUNT
+    // @DisplayName: Number of ADSB aircrafts
+    // @Description: Total number of ADSB simulated aircrafts
     AP_GROUPINFO("ADSB_COUNT",    45, SIM,  adsb_plane_count, -1),
+    // @Param: ADSB_RADIUS
+    // @DisplayName: ADSB radius stddev of another aircraft
+    // @Description: Simulated standard deviation of radius in ADSB of another aircraft
+    // @Units: m
     AP_GROUPINFO("ADSB_RADIUS",   46, SIM,  adsb_radius_m, 10000),
+    // @Param: ADSB_ALT
+    // @DisplayName: ADSB altitude of another aircraft
+    // @Description: Simulated ADSB altitude of another aircraft
+    // @Units: m
     AP_GROUPINFO("ADSB_ALT",      47, SIM,  adsb_altitude_m, 1000),
     AP_GROUPINFO("PIN_MASK",      50, SIM,  pin_mask, 0),
+    // @Param: ADSB_TX
+    // @DisplayName: ADSB transceiever enable
```

Comment:
```suggestion
    // @DisplayName: ADSB transmit enable
```

... it's the transmit part of the transceiver which is important here